rust-mode.el

```diff
@@ -1463,7 +1463,13 @@ This is written mainly to be used as `end-of-defun-function' for Rust."
              ((zerop ret)
               (if (not (string= (buffer-string)
                                 (with-current-buffer buf (buffer-string))))
-                  (copy-to-buffer buf (point-min) (point-max)))
+                  ;; replace-buffer-contents was in emacs 26.1, but it
+                  ;; was broken for non-ASCII strings, so we need 26.2.
+                  (if (and (fboundp 'replace-buffer-contents)
+                           (version<= "26.2" emacs-version))
```

Comment:
If I understand well, `(version<= "26.2" emacs-version)` should be enough, right?

Reply:
That's what I thought too :) But then the tests fail for emacs 24 and 25 because there is a call to an unbound function. 

https://github.com/tspiteri/rust-mode/actions/runs/88559854

```
Run ./run_rust_emacs_tests.sh
Byte-compilation failed:

In end of data:
rust-mode.el:2108:1:Warning: the function `replace-buffer-contents' is not
    known to be defined.
##[error]Process completed with exit code 4.
```

Reply:
Ok, if you don't know hacks to calm the byte compiler let's leave it as is.